packages/schematics/src/collection/ngrx/index.ts

```diff
@@ -30,6 +30,11 @@ import { formatFiles } from '../../utils/rules/format-files';
 export default function generateNgrxCollection(_options: Schema): Rule {
   return (host: Tree, context: SchematicContext) => {
     const options = normalizeOptions(_options);
+
+    if (!host.exists(options.module)) {
+      throw new Error('Specified module does not exist');
```

Comment:
Please specify the module name in the error. Otherwise the error message is simply not clear.

```ts
throw new Error(`Specified module '${options.module}' does not exist.`);
```